### DISCUSSION TITLE
When did toonsquid started progress in the making?

### DISCUSSION THREAD

**Discussion by VeiloPot** [keiwando/toonsquid#320] [General]
I was always wondering this, and I'm really interested to hear how long did you make it. Are you the only one that made it? Did you get any inspiration from it?

**Comment by Cat-Butler** [comment]
[ToonSquid FAQ](https://toonsquid.com/faq/): "ToonSquid is entirely made by one person, Keiwan Donyagard."

[Welcome post](https://github.com/keiwando/toonsquid/discussions/1) starting this forum: Jul 2, 2022
First [announcement on Twitter](https://twitter.com/toonsquid/status/1554519213267615745): Aug 3, 2022
[Reddit text](https://www.reddit.com/r/DigitalArt/comments/weiiue/i_made_a_2d_animation_app_for_the_ipad/): "I made this app (ToonSquid) in my spare time over the last few years in order to provide a high-quality animation tool for the iPad."

I don't immediately see any posts stating any definite sources of inspiration.
